selenium/src/test/java/au/org/emii/portal/tests/step1/FacetTest.java

```diff
@@ -139,10 +141,12 @@ private void verifyOrganisation(String org) {
             wait.until(ExpectedConditions.stalenessOf(results.get(0)));
             results = webElementUtil.findElements(By.className("resultsTextBody"));
         }
+        int expectedLastPageResults = resultsCount % 10;
+        if(expectedLastPageResults==0) {
+            expectedLastPageResults = 10;
+        }
```

Comment:
`int expectedLastPageResults = (resultsCount % 10 == 0) ? 10 : resultsCount % 10;`

How does that look?